Donation-platform/src/main/webapp/index.html

```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" href="personalizedStyle.css">
     <script src="userAuth.js"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"></script>
```

Comment:
include all third party scripts before putting any of your own scripts

Reply:
also for css files

Reply:
Done.